README.md

```diff
@@ -264,6 +264,10 @@ When developing, you can use the `Makefile` for doing the following operations:
 If you are working in a large organization you should learn more about [Kong
 Enterprise](https://konghq.com/kong-enterprise-edition/).
 
+## Contributions
+
+Contribution to this repository is always welcome. Please read [contributing.md](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md) , [Code-of-Conduct.md](https://github.com/Kong/kong/blob/master/CODE_OF_CONDUCT.md), [Copyrights.md](https://github.com/Kong/kong/blob/master/COPYRIGHT) and [Developer.md](https://github.com/Kong/kong/blob/master/DEVELOPER.md) before contributing.
```

Comment:
@Tieske Changes are done as per asked please check.

Reply:
no, you changed "COPYRIGHT" to "DEVELOPER.md" which is a nice catch, but the entire text should go in a different place, under the heading "Development" at line 132

Reply:
also, please remove the full urls, and replace them by the short markdown ones, for example: `[License](#license)` instead of the full url.